pkg/apis/topology/v1alpha2/types.go

```diff
@@ -0,0 +1,159 @@
+package v1alpha2
+
+import (
+	_ "github.com/gogo/protobuf/gogoproto"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type TopologyManagerPolicy string
+
+const (
+	// LabelAlignmentScope represents the scope hardware topology alignment performed on the node.
+	LabelAlignmentScope = "topology.node.k8s.io/alignment-scope"
+)
+
+const (
+	// AlignmentScopeContainer means this node is aligning resources at container level
+	AlignmentScopeContainer = "container"
+	// AlignmentScopeContainer means this node is aligning resources at pod level
+	AlignmentScopePod = "pod"
+)
+
+const (
+	// LabelAlignmentConstraint represents how strictly resources are aligned when allocated on the node.
+	// Failing to align the resources using the constraint will cause workload admission failure.
+	LabelAlignmentConstraint = "topology.node.k8s.io/alignment-constraint"
+)
+
+const (
+	// AlignmentConstraintBesteffort signals the node will try to align resources, but no baseline is
+	// enforced and if no alignment is possible the workload will be admitted anyway.
+	AlignmentConstraintBestEffort = "besteffort"
```

Comment:
```suggestion
	AlignmentConstraintBestEffort = "best-effort"
```

Reply:
I'm trying to use only single-word values, but I won't hold this position too strongly

Reply:
Just FYI in Polish if there is `-` between two words we consider this as a new single word 😄

Reply:
thinking about this, it's probably true also in english (and in italian FWIW)? Ok, let's just say  I'm trying to use values which fit in `[a-z][A-Z][0-9]*` but again, I won't hold this position strongly.